src/patterns/dosdp-patterns/nutrition_supplementation_by_chemical_role.yaml

```diff
@@ -9,10 +9,10 @@ contributors:
 
 classes:
   nutritional supplementation: MAXO:0000106
-  food material: FOODON:00002403
+  material entity: BFO:0000040
```

Comment:
The range of this pattern should not be material entity - this should be CHEBI:role! It wont change anything for the rest of the PR, but it will help with validation. The role pattern should only be used with roles, the normal supplement pattern with anything else.